lib/region/README.md

```diff
@@ -0,0 +1,38 @@
+# Region
+
+The `region` module is a module meant to standardize the detection of various
+regions, such as EU or APAC. It exposes functions that can test if a locale
+belongs to various regions. It can also simply lookup the region for a given
+locale.
+
+So far, the only region supported is EU, but the module is designed to be
+easily extendable when the need to support other regions pops up.
+
+## Usage Examples
+
+Loading the root module:
+```javascript
+var region = requrie('bv-ui-core/lib/region');
+
+region.has('en_US', 'europe'); // false
+region.has('en_GB', 'europe'); // true
+
+region.lookup('GB'); // 'europe'
+region.lookup('US'); // 'unsupported region'
+```
+This can be useful when you'll have to support detection of multiple regions.
+As only the EU region is supported for now, it's not particularly useful yet.
+
+Loading a specific region module:
+```javascript
+var europe = require('bv-ui-core/lib/region/europe');
+
+europe.has('en_US'); // false
+europe.has('en_GB'); // true
+
+// Get an Array with all the region's country codes.
+var europeanCountryList = europe.listCountryCodes();
```

Comment:
Further nitpickiness: I looked at [the spec](http://www.open-std.org/jtc1/sc22/wg20/docs/n610.pdf), and the language used indicates that a locale represents a "natural language" and a "territory". Since a territory is not necessarily a country, the use of "country" isn't completely semantically accurate.

I make no claim that you should change it, and I don't mean to build a bike shed; I'm particular just enough to share the findings :smiley:.